bin/anthology/papers.py

```diff
@@ -78,6 +79,16 @@ def pdf(self):
             return infer_url(url, template=data.PDF_LOCATION_TEMPLATE)
         return None
 
+    @cached_property
+    def videos(self):
+        videos = self.attrib.get("video", None)
+        if videos:
+            return [
+                infer_url(video, template=data.VIDEO_LOCATION_TEMPLATE)
+                for video in videos
+            ]
+        return []
+
     def _parse_revision_or_errata(self, tag):
```

Comment:
I think in `master` branch videos must be handled as general attachments, but I don't remember offhand. This is a good direction, though; you just have to figure out why it broke the current videos.